library/src/main/java/com/android/volley/Request.java

```diff
@@ -759,6 +759,8 @@ public long getTTL() {
 
     /**
      * Milliseconds to be considered 'fresh' in cache
+     * This means that if the current time is less then softttl expire time, the request can
```

Comment:
it can skip the network. We have a mechanism for warming the cache but the client won't get a response.